lib/engine/step/special_token.rb

```diff
@@ -94,7 +95,13 @@ def teleport_complete
       end
 
       def available_hex(entity, hex)
-        return if !ability(entity).hexes.empty? && !ability(entity).hexes.include?(hex.id)
+        ability = ability(entity)
+
+        if ability.respond_to?(:reachable) && ability.reachable
```

Comment:
Same as my other comment--but I do have another question, what other kinds of abilities can end up being called here that wouldn't be a `Token` ability that responds to `reachable`?

Reply:
Teleport IIRC ;)